### ISSUE TITLE
ELI5, having trouble installing

### ISSUE THREAD

**Issue by amingilani** [tip4commit/tip4commit#210]
Hi, i'm having trouble testing out an instance of tip4commit on a fresh Ubuntu 14.04.1 LTS box and this is what I've done so far:
1. Installed git, ruby, rubygems, rvm, dev tools
2. Clone repo
3. run `bundle install --without production` and hit an error

What am i doing wrong?

```
user@ubuntu:~/sandbox/tip4commit$ bundle install --without production
Fetching source index from https://rubygems.org/
Using rake 10.1.1
Using i18n 0.6.9
Using minitest 4.7.5
Using multi_json 1.8.4
Using atomic 1.1.14
Using thread_safe 0.1.3
Using tzinfo 0.3.38
Using activesupport 4.0.2
Using builder 3.1.4
Using erubis 2.7.0
Using rack 1.5.2
Using rack-test 0.6.2
Using actionpack 4.0.2
Using mime-types 1.25.1
Using polyglot 0.3.3
Using treetop 1.4.15
Using mail 2.5.4
Using actionmailer 4.0.2
Using activemodel 4.0.2
Using activerecord-deprecated_finders 1.0.3
Using arel 4.0.1
Using activerecord 4.0.2
Using acts_as_paranoid 0.5.0.beta1 from git://github.com/ActsAsParanoid/acts_as_paranoid.git (at master)
Using addressable 2.3.5
Using airbrake 3.1.15
Using bcrypt-ruby 3.1.2
Using bootstrap_form 2.2.0 from git://github.com/bootstrap-ruby/rails-bootstrap-forms.git (at master)
Using cancancan 1.7.1
Using net-ssh 2.7.0
Using net-scp 1.1.2
Using tins 0.13.1
Using term-ansicolor 1.2.2
Using sshkit 1.3.0
Using capistrano 3.0.1
Using capistrano-bundler 1.1.1
Using capistrano-rails 1.1.0
Using capistrano-rvm 0.1.0 from git://github.com/capistrano/rvm.git (at master)
Using mini_portile 0.5.3
Using nokogiri 1.6.1
Using xpath 2.0.0
Using capybara 2.2.1
Using coffee-script-source 1.6.3
Using execjs 2.0.2
Using coffee-script 2.2.0
Using thor 0.18.1
Using railties 4.0.2
Using coffee-rails 4.0.1
Using columnize 0.8.9
Using commonjs 0.2.7
Using diff-lcs 1.2.5
Using gherkin 2.12.2
Using multi_test 0.1.1
Using cucumber 1.3.14
Using bundler 1.7.6
Using hike 1.2.3
Using tilt 1.4.1
Using sprockets 2.10.1
Using sprockets-rails 2.0.1
Using rails 4.0.2
Using cucumber-rails 1.4.0
Using database_cleaner 1.2.0
Using debugger-linecache 1.2.0
Using debugger-ruby_core_source 1.3.5

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    /home/user/.rvm/rubies/ruby-2.0.0-p598/bin/ruby -r ./siteconf20150107-17163-1py2grz.rb extconf.rb 
checking for rb_method_entry_t.called_id in method.h... no
checking for rb_control_frame_t.method_id in method.h... no
checking for rb_method_entry_t.called_id in method.h... no
checking for rb_control_frame_t.method_id in method.h... no
Makefile creation failed
**************************************************************************
No source for ruby-2.0.0-p598 provided with debugger-ruby_core_source gem.
**************************************************************************
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
  --with-opt-dir
  --without-opt-dir
  --with-opt-include
  --without-opt-include=${opt-dir}/include
  --with-opt-lib
  --without-opt-lib=${opt-dir}/lib
  --with-make-prog
  --without-make-prog
  --srcdir=.
  --curdir
  --ruby=/home/user/.rvm/rubies/ruby-2.0.0-p598/bin/ruby
  --with-ruby-dir
  --without-ruby-dir
  --with-ruby-include=${ruby-dir}/include
  --with-ruby-lib
  --without-ruby-lib=${ruby-dir}/

extconf failed, exit code 1

Gem files will remain installed in /home/user/.rvm/gems/ruby-2.0.0-p598/gems/debugger-1.6.8 for inspection.
Results logged to /home/user/.rvm/gems/ruby-2.0.0-p598/extensions/x86_64-linux/2.0.0/debugger-1.6.8/gem_make.out
An error occurred while installing debugger (1.6.8), and Bundler cannot continue.
Make sure that `gem install debugger -v '1.6.8'` succeeds before bundling.

```

**Comment by amingilani** [comment]
Bumping this issue up for some help.

**Comment by dennisvandehoef** [comment]
the project is ruby 2.0.0, debugger is (quote) "Only 1.9.2 and 1.9.3 are supported. For 2.X rubies, consider using byebug."

https://github.com/cldwalker/debugger#known-issues
